recipes-multimedia/libcamera-apps/rpi-libcamera-apps_git.bb

```diff
@@ -0,0 +1,41 @@
+SUMMARY = "A suite of libcamera-based apps for the Raspberry Pi"
+DESCRIPTION = "This is a small suite of libcamera-based apps that aim to \
+copy the functionality of the existing \"raspicam\" apps."
+HOMEPAGE = "https://github.com/raspberrypi/libcamera-apps"
+SECTION = "console/utils"
+
+SRC_URI = "\
+    git://github.com/raspberrypi/libcamera-apps.git;protocol=https;branch=main \
+"
+SRCREV = "22a52590c33a813743b4e6337478c208201c77b1"
+
+LICENSE = "BSD-2-Clause"
+LIC_FILES_CHKSUM = "file://license.txt;md5=a0013d1b383d72ba4bdc5b750e7d1d77"
+
+S = "${WORKDIR}/git"
+
+COMPATIBLE_MACHINE = "^rpi$"
+
+DEPENDS = "libcamera libexif jpeg tiff libpng boost"
+RDEPENDS:${PN} = "python3-core libcamera libexif jpeg tiff libpng"
```

Comment:
I would expect the build system to pick up most of these. Any reason why that is not the case here?

Reply:
Are you referring to DEPENDS, RDEPENDS, or both? DEPENDS is needed otherwise I get build errors. RDEPENDS I am not so sure.

Reply:
I was only aiming the comment at RDEPENDS.

Reply:
Will check if build works without RDEPENDS.

Reply:
Build fails due to QA error

```log
ERROR: rpi-libcamera-apps-git-r0 do_package_qa: QA Issue: /usr/bin/camera-bug-report contained in package rpi-libcamera-apps requires /usr/bin/python3, but no providers found in RDEPENDS:rpi-libcamera-apps? [file-rdeps]
```

I can leave `RDEPENDS:${PN} = "python3-core"` or eliminate /usr/bin/camera-bug-report. Since not all systems have python, the latter sounds like a better option.

Reply:
Sounds good to me.